Apps/GatewayApi/src/Controllers/UserProfileController.cs

```diff
@@ -389,6 +392,24 @@ public RequestResult<UserProfileModel> UpdateAcceptedTerms(string hdid, [FromBod
             return result;
         }
 
+        /// <summary>
+        /// Test US phone number validation required by the GatewayAPI and PHSA.
+        /// </summary>
+        /// <param name="phoneNumber">Phone number stripped of any masked characters.</param>
+        /// <returns>True if the submitted phone number conforms to US standards.</returns>
+        [HttpGet]
+        [Route("IsValidPhoneNumber/{phoneNumber}")]
+        [AllowAnonymous]
```

Comment:
I think this should still require authorization to access, since it's currently only used in relation to verifying user SMS numbers.  We have just a few public endpoints and they have to be explicitly added to our Kong configurations to be accessible without logging in.

Reply:
Cool I'll make the change